build/generic/web/viewer.html

```diff
@@ -29,11 +29,13 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
 <link rel="stylesheet" href="<?= $viewerCSS ?>"> 
 
+<script defer src="//cdn.polyfill.io/v3/polyfill.min.js?features=default,Object.entries,Number.isInteger,IntersectionObserver,NodeList.prototype.forEach,WeakSet,WeakMap,fetch,Array.prototype.flatMap"></script>
```

Comment:
I'm bummed that this is going to be independent from our other polyfill includes, but I see it's hard to avoid with the submodule.